.travis.yml

```diff
@@ -13,3 +13,14 @@ sudo: false
 cache:
   directories:
     - $HOME/.yarn-cache
+deploy:
+  provider: releases
+  api_key:
+    secure: YeKIPlHkpDep5GB/VXzs2XyF2ym5Lh2nQ9XMsSntvxJ4K/MR0xRpwTxFN6TH1o4CW9RvlBGhhrWdeVGyMqEwnVY9OUx7jPkfKSkKBZZYx5DFMCuXxTJ0AgxT7IQjrsuweDdOWc5wdyKyxeJEJRnPfp4GOSPd8pJYSNKpJdmSzgM=
+  file:
+    - build/dist/uproxy-chrome-extension.zip
+    - build/dist/uproxy-chrome-app.zip
+    - build/dist/uproxy-chrome.zip
+    - build/dist/jid1-uTe1Bgrsb76jSA@jetpack-0.9.11.xpi
```

Comment:
This changes for every version, right? I guess you need to update the bump script.
Or maybe we should stop putting the version number in the file name.